db/migrations/0005.sql

```diff
@@ -0,0 +1,5 @@
+-- +migrate Up
+ALTER TABLE token ALTER COLUMN name TYPE varchar(30);
```

Comment:
Could you use the type `VARCHAR` (with no limits) so we avoid same problem in the future?